Code/Tools/TestImpactFramework/Runtime/Native/Code/Source/TestRunner/Native/Job/TestImpactNativeTestJobInfoUtils.cpp

```diff
@@ -31,4 +32,19 @@ namespace TestImpact
                 .c_str();
         }
     }
+
+    typename NativeTestEnumerator::Command GenerateTestEnumeratorJobInfoCommand(
+        const AZStd::string& launchArguement, const RepoPath& runArtifact)
+    {
+        const auto regularCommand = GenerateRegularTestJobInfoCommand(launchArguement, runArtifact);
+        return { AZStd::string::format("%s --gtest_list_tests", regularCommand.m_args.c_str()) };
+    }
+
+    typename NativeRegularTestRunner::Command GenerateRegularTestJobInfoCommand(
```

Comment:
Repeat: typename is not needed here.
```suggestion
    NativeTestEnumerator::Command GenerateTestEnumeratorJobInfoCommand(
        const AZStd::string& launchArguement, const RepoPath& runArtifact)
    {
        const auto regularCommand = GenerateRegularTestJobInfoCommand(launchArguement, runArtifact);
        return { AZStd::string::format("%s --gtest_list_tests", regularCommand.m_args.c_str()) };
    }

    NativeRegularTestRunner::Command GenerateRegularTestJobInfoCommand(
```